frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -169,6 +182,12 @@ export default {
     isSameUser() {
       return this.user.userName === this.userInfo.userName
     },
+    allFacilitiesSelected() {
+      return this.selectedFacilityIds.length === this.facilitiesToAdminister.length
+    },
+    someFacilitiesSelected() {
+      return this.selectedFacilityIds.length !== this.facilitiesToAdminister.length
```

Comment:
With the current logic, I think someFacilitiesSelected() is same as **!**allFacilitiesSelected(). Do we need to have both of them?

Reply:
FYI, in NewRequestDialog.vue, someFacilitiesSelected() checks if the array's length > 0

Reply:
copied the logic from newRequestDialog.